infra/gcp/terraform/k8s-infra-prow-build-trusted/prow-build-trusted/resources/test-pods/test-pods-serviceaccounts.yaml

```diff
@@ -31,6 +33,14 @@ metadata:
     iam.gke.io/gcp-service-account: k8s-triage@k8s-infra-prow-build-trusted.iam.gserviceaccount.com
   name: k8s-triage
   namespace: test-pods
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  annotations:
+    iam.gke.io/gcp-service-account: k8s-keps@k8s-infra-prow-build-trusted.iam.gserviceaccount.com
```

Comment:
Missing workload identity binding ?

Reply:
This is a common enough pattern that I opted to refactor into a module before adding yet more copy-pasta

I'll rebase and add the new SA/binding once https://github.com/kubernetes/k8s.io/pull/2574 merges